shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/presentation/uimodel/SurveyAnswerUiModel.kt

```diff
@@ -15,3 +16,9 @@ data class SurveyAnswerUiModel(
         placeholder = answer.placeholder
     )
 }
+
+fun SurveyAnswerUiModel.toAnswerSubmission() = AnswerSubmission(id, text)
+
+fun List<SurveyAnswerUiModel>.mapToAnswerSubmissions(): List<AnswerSubmission> {
```

Comment:
Same here. `map` is redundant IMO.
```suggestion
fun List<SurveyAnswerUiModel>.toAnswerSubmissions(): List<AnswerSubmission> {
```

Reply:
solved in 0318aa5